ci/nginx_72/docker-compose.yml

```diff
@@ -10,6 +10,8 @@ services:
   openemr:
     restart: always
     image: openemr/dev-php-fpm:7.2
+    environment:
```

Comment:
We'll use this environment variable to "override" using `http://localhost` in the e2e browser tests.

Reply:
nice!
(now it all makes sense since the php/nginx are in separate dockers :) )

Reply:
Yep! And since docker-compose is used the service name == the host name so there you go. Like all things it's obvious once you figure it out 😆